fleeto_backend/app/controllers/customer/v1/trip_requests_controller.rb

```diff
@@ -14,8 +14,7 @@ def create
 
   def destroy
```

Comment:
This should explicitly be named "canceled" and the service name should be TripRequets::CancelationService.
Destroy is kinda implicit.

This is just an opinion.